cmd/build.go

```diff
@@ -223,6 +239,10 @@ func runExecute(cmd *cobra.Command, args []string) error {
 		return errors.Wrap(err, "Could not find pwd")
 	}
 
+	if err = ensureDockerIsRunning(); err != nil {
```

Comment:
I might need to add this in more places 🤔

Reply:
Maybe to the proxy commands as well.